src/inspector_socket_server.cc

```diff
@@ -230,6 +242,7 @@ class SocketSession {
 
  private:
   enum class State { kHttp, kWebSocket, kClosing, kEOF, kDeclined };
+  // XXX(sam) why do these two methods have a trailing _? that's for properties
```

Comment:
Typo... AFAIR, they might've had _Io suffix at some point to signify they are called on IO thread.